### ISSUE TITLE
Build ignores --with-libxc-includedir=, --with-libxc-libs=, --with-hdf5-includedir=, --with-hdf5-libs= and still downloads tarballs

### ISSUE THREAD

**Issue by yurivict** [yambo-code/yambo#43]
Despite these being supplied build still downloads tarballs:
> ./configure CPP="gcc10 -E -P" FPP="gfortran10 -E -P -cpp" --with-netcdf-includedir="`pkg-config --cflags netcdf netcdf-fortran`" --with-netcdf-libs="`pkg-config --libs netcdf netcdf-fortran`"  --with-hdf5-includedir=/usr/local/include --with-hdf5-libs="-lhdf5"  --with-libxc-includedir="`pkg-config --cflags libxc libxcf90`" --with-libxc-libs="`pkg-config --libs libxc libxcf90`" --prefix=/usr/local

Additionally, it can't use external pnetcdf package and just downloads the tarball.

FYI: downloads aren't allowed by package builders. All dependencies should be either pre-installed as packages, or pre-downloaded.

Such difficulties likely cause porting systems to not have ports for Yambo.

I am trying to create a FreeBSD port but I just can't build Yambo.

Also after tarballs are downloaded build just fails ungracefully:
```
===>  Building for yambo-5.1.0
gmake[1]: Entering directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
gmake[2]: Entering directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
gmake[2]: warning: -j6 forced in submake: resetting jobserver mode.
gmake[2]: *** No rule to make target 'w'.  Stop.
gmake[2]: Leaving directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
yambo build failed
gmake[1]: Leaving directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
===>  Staging for yambo-5.1.0
===>   yambo-5.1.0 depends on executable: gfortran10 - found
===>   yambo-5.1.0 depends on executable: gcc10 - found
===>   Generating temporary packing list
gmake[1]: Entering directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
gmake[1]: *** No rule to make target 'install'.  Stop.
gmake[1]: Leaving directory '/disk-samsung/freebsd-ports/science/yambo/work/yambo-5.1.0'
*** Error code 2
```

**Comment by sangallidavide** [comment]
Dear @yurivict , I released a tentative fix into branch 5.2 for the ignored flags. Can you try it ?

For the "downloads not allowed".
Before the installation procedure (also before configure), you can run 'Make download'.

pnetcdf was added, but it is not used. We will remove it in the future.

**Comment by sangallidavide** [comment]
I close the present issue since I did not get any further reply
